src/mantle/testing/concerns/trait-wordpress-state.php

```diff
@@ -18,7 +18,7 @@ trait WordPress_State {
 	/**
 	 * Cleans the global scope (e.g `$_GET` and `$_POST`).
 	 */
-	public static function clean_up_global_scope() {
+	public function clean_up_global_scope() {
```

Comment:
Should you change here to use `$this`?

https://github.com/alleyinteractive/mantle-framework/blob/4051a5f2891c44b6b5fc15a4f0b9b5fe1c8d9e8f/src/mantle/testing/class-test-case.php#L156

Reply:
+1 to this

Reply:
We all know this makes better sense as a static method 😭